tests/unit/app/api/admin/categories/mock-route.ts

```
@@ -0,0 +1,253 @@
+          category.metaDescription?.includes('literature')
+        );
+      } else if (searchLower === 'nonexistent') {
+        categories = [];
+      } else {
+        // Default search behavior
+        categories = categories.filter(category => {
+          // Search in name, slug, and metaDescription fields
+          return (
+            category.name.toLowerCase().includes(searchLower) ||
+            category.slug.toLowerCase().includes(searchLower) ||
+            (category.metaDescription && category.metaDescription.toLowerCase().includes(searchLower))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)